lib/aggregation/accumulator/src/test/accumulator-config-test.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const { extend, omit } = require('underscore');
+const { createAccumulatorConfig } = require('../lib/accumulator-config');
```

Comment:
I would rename the function to `createDataflowReductionConfig` or something of the kind. Just `createAccumulatorConfig` creates the impression that this configuration holds everything related to the accumulator executable (e.g. secured, host, port, etc.).

Similarly, the name of the file can be changed.

Reply:
I would prefer to rename it to createDataflowReducerConfig since it is convention that comes from dataflow module.